pyglet_utils/config_loader.py

```diff
@@ -0,0 +1,26 @@
+import configparser
+import os
+
+
+class ConfigLoader(object):
+    def __init__(self):
+        self.config = configparser.ConfigParser()
+        self.config.read(os.path.join(os.getcwd(), "game_config.cfg"))
```

Comment:
I think we should consider making the file path a parameter, leave this a little more flexible.